router/dataplane.go

```diff
@@ -2160,11 +2160,6 @@ func (p *scionPacketProcessor) validateHopExpiry( /*@ ghost ubScionL []byte, gho
 	// @ defer fold acc(p.path.Mem(ubPath), R14)
 	// @ unfold acc(p.path.Base.Mem(), R15)
 	// @ defer fold acc(p.path.Base.Mem(), R15)
-	// (VerifiedSCION) Gobra is unable to verify permissions for `p.path.PathMeta.CurrINF`
-	// and `p.path.PathMeta.CurrHF` directly at the function call.
-	// To work around this limitation, these fields are first stored in temporary variables.
-	tmpCurrINF := p.path.PathMeta.CurrINF
-	tmpCurrHF := p.path.PathMeta.CurrHF
```

Comment:
I'm surprised this still shows up in the diff. Can you make sure this is based on the latest version of the packSCMP PR to make sure I am not reviewing old changes?

Reply:
Yes, this Pr is up-to-date. It was more convenient for me to fix this here rather than switching to the other Pr.